internal/controller/siteconfig_builder.go

```diff
@@ -29,3 +36,17 @@ func NewSiteConfigBuilder(pLog logr.Logger) *SiteConfigBuilder {
 
 	return &scBuilder
 }
+
+// GetConfigMap retrieves the configmap from cluster for a given TemplateRef
+func GetConfigMap(ctx context.Context, c client.Client, templateRef sc_v1alpha1.TemplateRef) (*corev1.ConfigMap, error) {
```

Comment:
This is really only one function call, not sure it really need to be it's own function.

But also it doesn't need to be capitalized if it's not being used outside the `controller` package.

Reply:
The function is used for rendering templates (next PR).